doc/source/configuration/monitoring.rst

```diff
@@ -78,3 +93,31 @@ on the overcloud hosts:
 
 SMART reporting should now be enabled along with a Prometheus alert for
 unhealthy disks and a Grafana dashboard called ``Hardware Overview``.
+
+Alertmanager and Slack
+======================
+
+StackHPC Kayobe configuration comes bundled with an array of alerts but does not
+enable any receivers for notifications by default. Various receivers can be
+configured for Alertmanager. Slack is currently the most common.
+
+To set up alerts for a Slack channel, you will need a webhook. `Create a new app
+<https://api.slack.com/apps/new>`__ in the workspace you want to add alerts to.
+From the Features page, toggle Activate incoming webhooks on. Click Add new
+webhook to workspace. Pick a channel that the app will post to, then click
+Authorise.
+
+Next, create a ``prometheus-alertmanager.yml`` file under
+``etc/kayobe/kolla/config/prometheus/``. An example config is stored in this
+directory. The example configuration uses two Slack channels. One channel
+receives all alerts while the other only receives alerts tagged as critical. It
+also adds a silence button to temporarily mute alerts. To use the example in a
+deployment, you will need to generate two webhook URLs, one for each channel.
+These should then be set as the ``slack_api_url`` and ``api_url`` for the
+regular and critical alerts channels respectively. Both URLs should be encrypted
```

Comment:
We could be more explicit here and tell people to put the webhooks in `kayobe/secrets.yml`. This follows existing convention.  Eg. 

```
secrets_slack_notification_channel_url: <some_webhook_url>
secrets_slack_critical_notification_channel_url: <some_other_webhook_url>
```

Reply:
I've moved things around and added this example